ui/apps/platform/src/Containers/ComplianceEnhanced/ClusterCompliance/ScanConfigs/ScanConfigDetailPage.tsx

```diff
@@ -0,0 +1,49 @@
+/* eslint-disable @typescript-eslint/no-unused-vars */
+import React, { useCallback } from 'react';
+import { generatePath, Link, useParams } from 'react-router-dom';
+import { Button, Divider, PageSection } from '@patternfly/react-core';
+
+import { complianceEnhancedScanConfigDetailPath } from 'routePaths';
+import useRestQuery from 'hooks/useRestQuery';
+import { getScanConfig } from 'services/ComplianceEnhancedService';
+import ViewScanConfigDetail from './ViewScanConfigDetail';
+
+const EditScanConfigButton = (id) => {
+    return (
+        <Link
+            to={`${generatePath(complianceEnhancedScanConfigDetailPath, {
+                scanConfigId: id,
+            })}?action=edit`}
+        >
+            <Button variant="primary">Edit scan schedule</Button>
```

Comment:
Will this be wired up to an edit box in a separate commit?

Reply:
Yes, this PR is just to get the read-only view working.